### PR TITLE
added test for bz1626093

### PR DESCRIPTION
**Pull request by kontura**
Testing whether plugin can successfully use DNF api to set http headers.
https://bugzilla.redhat.com/show_bug.cgi?id=1626093

### CODE REVIEW

File: dnf-docker-test/features/plugin-api-1.feature [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            repo.set_http_headers([ 
+                "custom_user_key20190218: custom_user_value", 
+            ])
+  """
+  Given a file "/etc/dnf/dnf.conf" with 
+  """
+  [main]
+  pluginpath=/test/plugins
+  """
+  And I successfully run "dnf clean all"
```
Comment (kkaarreell) [inline dnf-docker-test/features/plugin-api-1.feature]: When seems to be more appropriate than Given (And)

File: dnf-docker-test/features/plugin-api-1.feature [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  Then I sniff packets expecting "custom_user_key20190218: custom_user_value"
+  Given a file "/etc/dnf/dnf.conf" with
```
Comment (kkaarreell) [inline dnf-docker-test/features/plugin-api-1.feature]: What is the purpose of this Given? Is this a clean up step (which is not necessary)? Or is the test somehow incomplete?

Comment (kontura) [reply dnf-docker-test/features/plugin-api-1.feature]: It was supposed to be clean up, I originally had it in different feature with other tests. You are right, it is not necessary.

**Comment by kontura** [comment]
Thank you for feedback. I updated the commit.

**Pull request merged**